workflow/hydrator/hydrator.go

```diff
@@ -85,6 +85,7 @@ func (h hydrator) Hydrate(wf *wfv1.Workflow) error {
 		}
 		h.HydrateWithNodes(wf, offloadedNodes)
 	}
+	log.WithField("Workflow Size", wf.Size()).Info("Workflow hydrated")
```

Comment:
can you move to in  86 after h.HydrateWithNodes?

Reply:
yes